connection.js

```diff
@@ -169,8 +172,10 @@ function Connection(client, server) {
     this.transaction = null;
     this.tran_count = 0;
     this.capabilities = null;
-    this.early_talker_delay = config.get('early_talker.pause') || config.get('early_talker_delay') || 1000;
-    this.banner_includes_uuid = config.get('banner_includes_uuid') ? true : false;
+    this.early_talker_delay = config.get('early_talker.pause') ||
+                              config.get('early_talker_delay') || 1000;
+    this.banner_includes_uuid =
+        config.get('banner_includes_uuid') ? true : false;
```

Comment:
I believe what you're asking for is implemented [right here](https://github.com/baudehlo/Haraka/blob/master/configfile.js#L217).